server/services/interventionsService.test.ts

```diff
@@ -991,3 +1051,41 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     })
   })
 })
+
```

Comment:
do these new contracts already pass or is there any work required on backend for these?

Reply:
comment on wrong part of file- but i'm talking about the contract test above